server/apps/frontend/src/Seller/SellerForm/SellerFrom.hook.ts

```diff
@@ -0,0 +1,57 @@
+import React, { FormEvent, RefObject } from 'react';
+import { useMutation } from 'react-query';
+import { AddSellerType, SellerForm } from '../Seller.hook';
+
+export const useForm = (
+  addSeller: AddSellerType
+): {
+  nameRef: RefObject<HTMLInputElement>;
+  passwordRef: RefObject<HTMLInputElement>;
+  urlRef: RefObject<HTMLInputElement>;
+  handleSubmit: (event: FormEvent) => void;
+} => {
+  const nameRef: RefObject<HTMLInputElement> = React.useRef(null);
+  const passwordRef: RefObject<HTMLInputElement> = React.useRef(null);
+  const urlRef: RefObject<HTMLInputElement> = React.useRef(null);
+
+  const addSellerMutation = useMutation((newSeller: SellerForm) => {
+    return fetch('/seller', {
+      method: 'POST',
+      body: new URLSearchParams(newSeller),
+    });
+  });
+
+  const handleSubmit = (event: FormEvent) => {
+    const name = nameRef.current?.value;
+    const password = passwordRef.current?.value;
+    const url = urlRef.current?.value;
+
+    event.preventDefault();
+    if (!name || !url) {
```

Comment:
C'est un bug existant, le front autorise un mot de passe vide mais pas le backend.

```suggestion
    if (!name || !url || !password) {
```